ddspipe_core/src/cpp/configuration/RoutesConfiguration.cpp

```diff
@@ -101,6 +101,89 @@ RoutesConfiguration::RoutesMap RoutesConfiguration::operator ()() const
     return routes;
 }
 
+RoutesConfiguration::RoutesMap RoutesConfiguration::routes_of_readers(
```

Comment:
Change the name of this methods to show more clearly what they do. Try something like ``writers_in_the_route_of_readers``.